theforeman.org/pipelines/test/smart-proxy.groovy

```diff
@@ -79,6 +84,10 @@ pipeline {
                             script {
                                 git_checkout()
                             }
+
+                            dir('smart-proxy') {
+                                git url: "https://github.com/theforeman/smart-proxy", branch: develop, poll: false
+                            }
```

Comment:
I still fail to see the need for this. AFAIK plugins list smart-proxy as a dependency in `Gemfile` so it shouldn't be needed to check this out. In case of the Smart Proxy itself you just end up with a duplicate directory.
```suggestion
```